scipy/optimize/_highs/setup.py

```diff
@@ -72,11 +75,13 @@ def configuration(parent_package='', top_path=None):
         EXTRA_COMPILE_ARGS.append('-std=c++14')
 
     # -O3 takes longer to compile, but passes unit tests -- could use?
-    #EXTRA_COMPILE_ARGS.append('-O3')
+    # EXTRA_COMPILE_ARGS.append('-O3')
+    EXTRA_COMPILE_ARGS = ['-std=c++14']  # '-O3']
```

Comment:
Is this right? Probably should have removed the line above.

Reply:
I haven't done any testing to see if performance is better for large problems using -O3 optimizations and compared it against the increase in size of the binary/longer compilation times

Reply:
I think we should be consistent with HiGHS.  After snooping around generated Makefiles, I find that:
```
libbasiclu: -O3
libipx: -O3
libhighs: -O3
app/highs: -O3
```

Everything uses `-O3`, so I'll make that change.